keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -217,6 +229,7 @@ def __init__(
         if keep_pattern is None:
             keep_pattern = PUNCTUATION_REGEX
 
+        self._vocabulary_size = _vocabulary_size
```

Comment:
This line can be `self._vocabulary_size = vocabulary_size` to avoid the collision with the method of the same name.